25/documentation.html

```diff
@@ -31,7 +31,7 @@
     <!--#include virtual="toc.html" -->
   </div>
 	<div class="right">
-    <!--#include virtual="../includes/_docs_banner.htm" -->
```

Comment:
Ditto for other 25/xyx.html pages: does the same issue exist in older versions (24, 23...)?

Reply:
I believe the older versions SHOULD have this banner.